data/sandramsc.json

```diff
@@ -4,22 +4,26 @@
   "tags": [
     "React.js",
     "Python",
-    "Typescript",
-    "WordPress",
     "Open Source",
     "Artificial Intelligence (AI)",
+    "WordPress",
     "Web3"
   ],
   "socials": [
     {
       "icon": "FaLinkedin",
```

Comment:
[Proofreading] the `icon` string is not supposed to have a comma at the end.
```suggestion
      "icon": "FaLinkedin"
```

Reply:
Hi @CBID2 thanks for the feedback. I'm a tad confused as I did not make any changes to the icon strings from a previously merged commit a few months ago (simply duplicated a social for the twitter addition). Is this a recent design preference?

![Screenshot_20230621-015359](https://github.com/EddieHubCommunity/LinkFree/assets/19821445/76517088-1106-46d4-bc7c-853f0f64b020)

Reply:
Yeah pretty much @sandramsc

Reply:
@CBID2 this incorrect, removing this comma will make the json invalid and break the profile. I will mark this as resolved to avoid confusion